### ISSUE TITLE
About NaN

### ISSUE THREAD

**Issue by wxxhaoshuai** [SkyworkAI/MoH#3]
Hello,have you ever meet NaN of the output of the logits(self.x)

**Comment by jpthu17** [comment]
Hello. Have you checked whether x has a value? If the input value of x is NaN, then the output will also be NaN.
